src/components/select-project/select-project.controller.ts

```diff
@@ -0,0 +1,93 @@
+import * as angular from 'angular';
+import * as _ from 'lodash';
+
+export class SelectProjectController implements angular.IController {
+  static $inject = ['$scope', '$filter', 'DataService', 'ProjectsService', 'Logger', 'AuthService'];
+
+  public ctrl: any = this;
+  public $scope: any;
+
+  private $filter: any;
+  private DataService: any;
+  private ProjectsService: any;
+  private Logger: any;
+  private AuthService: any;
+
+  constructor($scope: any, $filter: any, DataService: any, ProjectsService: any, Logger: any, AuthService: any) {
+    this.$scope = $scope;
+    this.$filter = $filter;
+    this.DataService = DataService;
+    this.AuthService = AuthService;
+    this.ProjectsService = ProjectsService;
+    this.Logger = Logger;
+  }
+
+  public $onInit() {
+    this.ctrl.nameTaken = false;
+    // TODO disableInputs?
+    this.ctrl.disableInputs = false;
```

Comment:
I don't think this is needed, or at least it should be on the entire form, not just the select project control.